logistics/preparation.md

```diff
@@ -0,0 +1,109 @@
+        - Existing subscriptions
+        - Visual Studio subscriptions
+        - Existing dev/test (sandbox) subscriptions
+    - Collaborate with the preparation owner to determine the number of subscriptions required. Considerations:
+        - Number of teams or resources 
+        - **Recommendation**: Use a single sandbox subscription for all participants. Benefits of this approach include:
+            - Consistent Monitoring: Easier to track usage and costs across the board.
+            - Simplified Management: Reduces the administrative overhead of managing multiple subscriptions.
+            - Shared Resources: Facilitates seamless sharing of resources and information among teams.
+            - Cost Efficiency: Potentially lower costs as there are volume-based price benefits.
+    - Based on the chosen approach, define AD groups per team and create separate resource groups for each.
+    - Ensure participants have contributor access within their [specific resource groups](https://learn.microsoft.com/en-us/azure/role-based-access-control/quickstart-assign-role-user-portal).
+1. Request Access to Azure OpenAI Service for All Subscriptions    
+    - Visit [https://aka.ms/oai/access](https://aka.ms/oai/access). Submit details for each subscription.
+    - Monitor request status.   
+    **Note:** If your team requires other [limited access services](https://learn.microsoft.com/en-us/azure/ai-services/cognitive-services-limited-access) for the hack, you will need to request access for them.
+1. Create Shared Azure OpenAI Service Resources
+    - Initialize Azure OpenAI resources within each subscription
+    - Provision necessary resources (e.g., computation, storage)
+    - Document resource specifics for team access
+
+1. Deploy Models with Tokens Per Minute (TPM) Limits
+    - Ascertain model requirements for each team.
+    - [Deploy models](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource?pivots=web-portal) via Azure OpenAI Service, setting [TPM to prevent overuse](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/quota?tabs=rest#assign-quota).
+    **Note on TPM:** AOAI has limits on tokens per minutes for each subcription. These tokens are shared across all models of similar types in a region. Learn more about the limits [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/quotas-limits). This limit is crucial for hackathons with multiple teams. If one team consumes the entire quota, others will be left without access. Therefore:
+    - [Even Distribution](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/quota?tabs=rest#assign-quota): Consider dividing the TPM evenly among teams to ensure fair use.
+    - Monitoring: Regularly monitor token consumption to avoid any team exhausting the entire limit.
+    - Alerts: Set up notifications when a certain percentage of the TPM is reached to ensure corrective actions can be taken in time.  
```

Comment:
Not sure we can actually do this. But we should provide a link t show how many TPM have been assigned. See the AOAI Quota page.